data_mapping.bal

```diff
@@ -0,0 +1,26 @@
+
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+isolated function mapJsonToWatchResponse(map<json> jsonPayload) returns WatchResponse|error {
+    WatchResponse response = {};
+    response.kind = jsonPayload["kind"].toString();
+    response.id = jsonPayload["id"].toString();
+    response.resourceId = jsonPayload["resourceId"].toString();
+    response.resourceUri = jsonPayload["resourceUri"].toString();
+    response.expiration = check int:fromString(jsonPayload["expiration"].toString());
```

Comment:
Instead of creating an empty {} and assigning values later, we can do like this.
```
WatchResponse response = {
    kind: jsonPayload["kind"].toString(),
    id: jsonPayload["id"].toString(),
    ----
    ----
}

Reply:
Thanks @LakshanSS 
Changed that way